src/test/java/bio/terra/common/auth/AuthService.java

```diff
@@ -48,17 +50,22 @@
   private String saEmail;
   private Map<String, GoogleCredential> userTokens =
       Collections.synchronizedMap(new PassiveExpiringMap<>(TOKEN_CACHE_EXPIRATION_POLICY));
+  private Map<String, String> petAccountTokens =
+      Collections.synchronizedMap(new PassiveExpiringMap<>(TimeUnit.MINUTES.toMillis(55)));
```

Comment:
Another way to write this is
```suggestion
      Collections.synchronizedMap(new PassiveExpiringMap<>(55, TimeUnit.MINUTES));
```